core/src/tangram.cpp

```diff
@@ -672,17 +672,18 @@ void Map::setPixelScale(float _pixelsPerPoint) {
 
 void Map::Impl::setPixelScale(float _pixelsPerPoint) {
 
+    view.setPixelScale(_pixelsPerPoint);
```

Comment:
This guarantees that `_pixelsPerPoint == view.pixelScale()` below is always true, so the tile clearing steps are unreachable.